content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md

```diff
@@ -4,7 +4,7 @@ reviewers:
 - wojtek-t
 title: Pod Priority and Preemption
 content_type: concept
-weight: 70
+weight: 50
 ---
 
 <!-- overview -->
```

Comment:
Add mentions of the other types of disruption? It's fine for a follow up PR.

Reply:
I linked to it in a whatsnext section.